bot/exts/moderation/modpings.py

```diff
@@ -37,19 +37,25 @@ def __init__(self, bot: Bot):
         self.bot = bot
         self._role_scheduler = Scheduler("ModPingsOnOff")
         self._modpings_scheduler = Scheduler("ModPingsSchedule")
+        self.fetch_guild_task = self.bot.loop.create_task(self.fetch_guild())
```

Comment:
This pattern of making a task and awaiting it in functions isn't needed anymore, since d.py added async cog_load loading. As it guarantees that the cog_load function is finished before the cog accepts any command/events.

So this task, an all usages of it can be safely deleted.